backend/src/routes/user.ts

```diff
@@ -148,10 +149,35 @@ router.get("/mentee/:userId", [verifyAuthToken], async (req: Request, res: Respo
       if (!mentee) {
         throw ServiceError.MENTEE_WAS_NOT_FOUND;
       }
+      const {
+        _id,
```

Comment:
Think you could rename this variable to `menteeId` to fix linting by doing - 
```
_id: menteeId
```

Reply:
Yeah, or @AmanKAggarwal feel free to disable the `@typescript-eslint/naming-convention` lint rule too - I think this might have been added in a newer version of typescript-eslint, because I don't remember having this issue before.

Reply:
addressed